inc/savedsearch.class.php

```diff
@@ -595,33 +584,19 @@ function getParameters($ID) {
       return false;
    }
 
-   /**
-    * Mark saved search as default view for the current user
-    *
-    * @param integer $ID ID of the saved search
-    *
-    * @deprecated since version 9.2; @see SavedSearch::markDefault()
-    *
-    * @return void
-   **/
-   function mark_default($ID) {
```

Comment:
It's not a deprecated method because this method doesn't exist in current version.
I think you create this function and after change this name. Savedsearch.class doesn't exist in 9.1.5